packages/vue/src/components/BlockCsrTable/BlockCsrTable.stories.js

```diff
@@ -0,0 +1,220 @@
+import BlockCsrTable from './BlockCsrTable.vue'
+
+export default {
+  title: 'Components/FeatureFlags/BlockCsrTable',
+  component: BlockCsrTable,
+  excludeStories: /.*Data$/,
+  parameters: {
+    a11y: {
+      element: 'body',
+      config: {
+        rules: [
+          // ignore rules that fail tests but are not actually violations
+          // see: https://github.com/ag-grid/ag-grid/issues/6218
+          {
+            id: 'aria-required-children',
+            enabled: false
+          },
+          {
+            id: 'region',
```

Comment:
I had to change this to `'scrollable-region-focusable'` to get that error to go away, but then the actual `'region'` error ("All page content should be contained by landmarks") started appearing, and I'm not sure why that error doesn't appear on every other story, since it's never been disabled before.